src/hs13.jl

```diff
@@ -15,10 +15,10 @@ export hs13
 function hs13(args...)
 
   model = Model()
-  @variable(model, x[1:2] >= 0.0, start=-2.0)
+  @variable(model, x[1:2] ≥ 0.0, start=-2.0)
```

Comment:
It's a good idea to replace those, but there are more that you didn't change. I think they should be changed in another PR as it's unrelated to the other changes.

Reply:
I modified some of them because they were near the `setvalue` that I changed, but I only did a few files.
I remembered that I can use a sed call for that... ...in an other pull request.

Reply:
Let's keep PRs focused.